docs/src/content/docs/fr/reference/configuration.mdx

```diff
@@ -521,3 +531,18 @@ starlight({
 ```
 
 Consultez la [référence des redéfinitions](/fr/reference/overrides/) pour plus de détails sur tous les composants que vous pouvez redéfinir.
+
+### `plugins`
+
+**type:** [`StarlightPlugin[]`](/fr/reference/plugins/#référence-rapide-de-lapi)
```

Comment:
This link is currently invalid. I'll update this comment when fixed.

Reply:
#1195 is now merged! 🥳